api/setup.cfg

```diff
@@ -1,7 +1,2 @@
-[pycodestyle]
-max_line_length=120
-exclude=migrations
-
-[isort]
-line_length=120
-multi_line_output=5
+[flake8]
+max-line-length = 88
```

Comment:
Any specific reason for 88? I'm pro shorter lines, and that seems a fine length to me, just curious.

Reply:
I was following this black guide where it was suggested to match flake8 line length with that of black: https://github.com/psf/black#line-length